.travis.yml

```diff
@@ -1,3 +1,10 @@
 language: ruby
 rvm:
-    - 2.2.1
+  - 2.2.1
+
+before_script:
+    - psql -c 'create database ebay_clone_test;' -U postgres
```

Comment:
Isn't this what `rake db:create` does? I don't quite understand this file. I think the test key down below is config that should go in `config/database.yml`.